Pinta.Core/Classes/BaseTool.cs

```diff
@@ -69,6 +69,12 @@ protected BaseTool (IServiceManager services)
 		/// </summary>
 		public abstract string Icon { get; }
 
+		/// <summary>
+		/// The localized name of the tool. This can be overridden if custom translation
+		/// support is needed.
+		/// </summary>
+		public virtual string LocalizedName => Translations.GetString (Name);
```

Comment:
I think this default implementation will probably cause issues for extracting translatable strings with `xgettext` (`make updatepot`), which looks for string literals in the arguments to `GetString()`

Reply:
Hmm, that's unfortunate.  I guess it could be `abstract` and each tool will need to implement it.

Reply:
Yeah, I think it'll need to be implemented by each tool. At the same time, maybe we should also just make the `Name` clearly different from the localized label? (e.g. `move-selected-pixels` vs `Move Selected Pixels`) so that any mixup between the two can be caught on an English locale?

Reply:
I changed `BaseTool.Name` back to being the localized version, and instead changed `ToolsManager.SetCurrentTool (string)` and related to work on type names instead of the user visible names.  This should give us the non-localized tool identifier string we need while not forcing tools to do both `Name` and `LocalizedName`.